### DISCUSSION TITLE
远程服务器返回错误: (502) 错误的网关。

### DISCUSSION THREAD

**Discussion by Eric-Zhang007** [ScoopInstaller/Scoop#5877] [Help and Questions]
如题，请求大佬援助
scoop update *

sublime-text: 4-4150 -> 4-4169
utools: 3.3.0 -> 4.4.1
Updating 2 outdated apps:
Updating 'sublime-text' (4-4150 -> 4-4169)
Downloading new version
远程服务器返回错误: (502) 错误的网关。
所在位置 D:\Base\apps\scoop\current\lib\install.ps1:111 字符: 9
+         throw $e
+         ~~~~~~~~
    + CategoryInfo          : OperationStopped: (:) [], WebException
    + FullyQualifiedErrorId : 远程服务器返回错误: (502) 错误的网关。

**Comment by shoucandanghehe** [comment]
Check your network and the bucket that provides the sublime-text package
检查你的网络和提供sublime-text包的桶

**Comment by Eric-Zhang007** [comment]
however I still don't know how to solve it...
how can i skip this package?
